### ISSUE TITLE
if one entity has multiple mentions,how to deal this situation?

### ISSUE THREAD

**Issue by hitxujian** [princeton-nlp/SimCSE#51]
Thanks for sharing you code! in the paper you use cosine cross entropy as loss function 
![image](https://user-images.githubusercontent.com/11830865/123951306-3bd5de80-d9d7-11eb-87a9-ca313a0fe9c3.png)
and the code in the file models.py
`loss_fct = nn.CrossEntropyLoss()

    # Calculate loss with hard negatives
    if num_sent == 3:
        # Note that weights are actually logits of weights
        z3_weight = cls.model_args.hard_negative_weight
        weights = torch.tensor(
            [[0.0] * (cos_sim.size(-1) - z1_z3_cos.size(-1)) + [0.0] * i + [z3_weight] + [0.0] * (z1_z3_cos.size(-1) - i - 1) for i in range(z1_z3_cos.size(-1))]
        ).to(cls.device)
        cos_sim = cos_sim + weights

    loss = loss_fct(cos_sim, labels)`
but if the input is <mention,entity> pair,and one entity may has multiple mentions,so  one entiy may have multiple mentions labeled 1,how to process this situation? looking forward you answer

**Comment by gaotianyu1350** [comment]
So you mean there are multiple "positive pairs" for one instance? In that case, you can simply add all the positive instances to the nominator.
